microscope/monitor/runner.py

```diff
@@ -0,0 +1,300 @@
+from typing import List, Set, Callable, Dict
+import json
+import sys
+from multiprocessing import Queue
+
+from kubernetes import client
+from kubernetes.client.apis import core_v1_api
+from kubernetes.client.rest import ApiException
+from kubernetes.stream import stream
+
+from microscope.monitor.monitor import Monitor
+
+
+class MonitorArgs:
+    def __init__(self,
+                 verbose: bool,
+                 hex: bool,
+                 related_selectors: List[str],
+                 related_pods: List[str],
+                 related_endpoints: List[int],
+                 to_selectors: List[str],
+                 to_pods: List[str],
+                 to_endpoints: List[int],
+                 from_selectors: List[str],
+                 from_pods: List[str],
+                 from_endpoints: List[int],
+                 types: List[str],
+                 namespace: str,
+                 raw: bool
+                 ):
+        self.verbose = verbose
+        self.hex = hex
```

Comment:
`hex` is a builtin function in Python - even Github is coloring it blue. :) I'd suggest to change that name, otherwise any linter is going to complain.

Reply:
The variable name shadows the builtin, but I guess you're right. flake8 doesn't complain about it though.